src/pages/home/ReportScreen.js

```diff
@@ -228,8 +228,7 @@ class ReportScreen extends React.Component {
                 )}
             >
                 <ScreenWrapper
-                    style={screenWrapperStyle}
-                    keyboardAvoidingViewBehavior={Platform.OS === 'android' ? '' : 'padding'}
+                    style={[styles.appContent, styles.flex1, {marginTop: this.state.viewportOffsetTop}]}
```

Comment:
Why was this change made?
Why shouldn't we modify the `screenWrapperStyle`  variable instead?

Reply:
It was made to fix the spacing bug you pointed out. I'm not sure about the `screenWrapperStyle`, I was not aware of that prop. Is there a problem with this?

Reply:
Which issue are you referring, I pointed out on Login, it doesn't seem relevant?

Reply:
Oh, sorry! I got the components mixed up. I get what you're saying now :D You're right, I've changed this to modify the `screenWrapperStyle` now.